docs/configuration/configuration.md

```diff
@@ -18,98 +18,97 @@ An example [oauth2_proxy.cfg](contrib/oauth2_proxy.cfg.example) config file is i
-  -proxy-websockets: enables WebSocket proxying (default true)
-  -pubjwk-url string: JWK pubkey access endpoint: required by login.gov
-  -redeem-url string: Token redemption endpoint
-  -redirect-url string: the OAuth Redirect URL. ie: "https://internalapp.yourcompany.com/oauth2/callback"
-  -redis-connection-url string: URL of redis server for redis session storage (eg: redis://HOST[:PORT])
-  -redis-sentinel-master-name string: Redis sentinel master name. Used in conjuction with --redis-use-sentinel
-  -redis-sentinel-connection-urls: List of Redis sentinel conneciton URLs (eg redis://HOST[:PORT]). Used in conjuction with --redis-use-sentinel
-  -redis-use-sentinel: Connect to redis via sentinels. Must set --redis-sentinel-master-name and --redis-sentinel-connection-urls to use this feature (default: false)
-  -request-logging: Log requests to stdout (default true)
-  -request-logging-format: Template for request log lines (see "Logging Configuration" paragraph below)
-  -resource string: The resource that is protected (Azure AD only)
-  -scope string: OAuth scope specification
-  -session-store-type: Session data storage backend (default: cookie)
-  -set-xauthrequest: set X-Auth-Request-User and X-Auth-Request-Email response headers (useful in Nginx auth_request mode)
-  -set-authorization-header: set Authorization Bearer response header (useful in Nginx auth_request mode)
-  -signature-key string: GAP-Signature request signature key (algorithm:secretkey)
-  -silence-ping-logging bool: disable logging of requests to ping endpoint (default false) 
-  -skip-auth-preflight: will skip authentication for OPTIONS requests
-  -skip-auth-regex value: bypass authentication for requests path's that match (may be given multiple times)
-  -skip-jwt-bearer-tokens: will skip requests that have verified JWT bearer tokens
-  -skip-oidc-discovery: bypass OIDC endpoint discovery. login-url, redeem-url and oidc-jwks-url must be configured in this case
-  -skip-provider-button: will skip sign-in-page to directly reach the next step: oauth/start
-  -ssl-insecure-skip-verify: skip validation of certificates presented when using HTTPS providers
-  -ssl-upstream-insecure-skip-verify: skip validation of certificates presented when using HTTPS upstreams
-  -standard-logging: Log standard runtime information (default true)
-  -standard-logging-format string: Template for standard log lines (see "Logging Configuration" paragraph below)
-  -tls-cert-file string: path to certificate file
-  -tls-key-file string: path to private key file
-  -upstream value: the http url(s) of the upstream endpoint or file:// paths for static files. Routing is based on the path
-  -validate-url string: Access token validation endpoint
-  -version: print version string
-  -whitelist-domain: allowed domains for redirection after authentication. Prefix domain with a . to allow subdomains (eg .example.com)
-```
+| Option | Type | Description | Default |
+| ------ | ---- | ----------- |
```

Comment:
should this: `| ------ | ---- | ----------- |`
be this?: `| ------ | ---- | ----------- | ------- |`

Reply:
Good catch, fixed!